tests/lib/rules/block-scope-case.test.js

```diff
@@ -43,10 +56,9 @@ invalidExample2 = formatCode(
    '}'
 );
 
-
 ruleTester.run('block-scope-case', rule, {
    valid: [
-      validExample,
+      validExample1, validExample2,
```

Comment:
`validExample2` should be on it's own line. All values should be on their own line in a multiline array statements.

Reply:
good call